scipy/optimize/_differentialevolution.py

```diff
@@ -400,7 +408,9 @@ def __init__(self, func, bounds, args=(),
                                  self.parameter_count)
 
         self._nfev = 0
-        if init == 'latinhypercube':
+        if isinstance(init, np.ndarray):
```

Comment:
I think you should test for strings and assume array-like otherwise (such as list of lists would be a valid candidate if passed through `np.array(...)` ). It is annoying when the argument is rejected though it is a valid array but not specifically a Numpy array.